CHANGES/739.bugfix

```diff
@@ -0,0 +1,3 @@
+Create 'inbound-namespaces' whenever a namespace is created.
+
+This includes namespaces created via sync.
```

Comment:
recommend changing this changelog entry to say
`Ensure 'inbound-<namespace>' repo exists when accessing namespace`
since we already create these repos on sync - this just ensures namespaces sync'd before https://github.com/ansible/galaxy_ng/commit/8c5f3f945435a792504adc97df7bd2d2a94b412d get a repo